hvac_services/data/ir_model_fields.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="x_maintained_serials_on_sale_order_line_field" model="ir.model.fields">
+        <field name="name">x_maintained_serials_ids</field>
+        <field name="ttype">many2many</field>
+        <field name="field_description">Serials</field>
+        <field name="model_id" ref="sale.model_sale_order_line"/>
+        <field name="relation">stock.lot</field>
+        <field name="copied" eval="True"/>
```

Comment:
I wonder about those copied values... If we duplicate a sale order line, do we want to keep the serials? (real question)
Could you check other ones in the file and see if relevant or not?